virt/logging_events_monitoring/virt-collecting-virt-data.adoc

```diff
@@ -1,25 +1,54 @@
 
-include::modules/about-must-gather.adoc[leveloffset=+1]
+Alertmanager::
+The Alertmanager service handles alerts received from Prometheus. The Alertmanager is also responsible for sending the alerts to external notification systems.
 
-include::modules/virt-about-collecting-virt-data.adoc[leveloffset=+1]
+include::modules/virt-collecting-data-about-your-environment.adoc[leveloffset=+1]
 
-include::modules/gathering-data-specific-features.adoc[leveloffset=+1]
+[id="additional-resources_collecting-data-about-your-environment"]
+[role="_additional-resources"]
+=== Additional resources
 
-include::modules/virt-must-gather-usage-targeted-vm-data.adoc[leveloffset=+1]
+* Configuring the xref:../../monitoring/configuring-the-monitoring-stack.adoc#modifying-retention-time-for-prometheus-metrics-data_configuring-the-monitoring-stack[retention time] for Prometheus metrics data.
+* Configuring the Alertmanager to send xref:../../monitoring/managing-alerts.adoc#sending-notifications-to-external-systems_managing-alerts[alert notifications] to external systems.
+* Collecting `must-gather` data for xref:../../support/gathering-cluster-data.adoc#support_gathering_data_gathering-cluster-data[{product-title}].
+* Collecting `must-gather` data for link:https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.10/html-single/troubleshooting_openshift_data_foundation/index#downloading-log-files-and-diagnostic-information_rhodf[{rh-storage-first}].
+* Collecting `must-gather` data for xref:../../virt/logging_events_monitoring/virt-collecting-virt-data.adoc#gathering-data-specific-features_virt-collecting-virt-data[{VirtProductName}].
+* Collecting Prometheus metrics for xref:../../monitoring/managing-metrics.adoc#querying-metrics-for-all-projects-as-an-administrator_managing-metrics[all projects] as a cluster administrator.
+
+include::modules/virt-collecting-data-about-vms.adoc[leveloffset=+1]
+
+[id="additional-resources_collecting-data-about-vms"]
+[role="_additional-resources"]
+=== Additional resources
+
+* Installing xref:../../virt/virtual_machines/virt-installing-qemu-guest-agent.adoc#virt-installing-virtio-drivers-existing-windows_virt-installing-qemu-guest-agent[VirtIO drivers] on Windows VMs.
+* Downloading and installing link:https://access.redhat.com/solutions/6957701[VirtIO drivers] on Windows VMs without host access.
+* Connecting to Windows VMs with RDP using the xref:../../virt/virtual_machines/virt-accessing-vm-consoles.adoc#virt-vm-rdp-console-web_virt-accessing-vm-consoles[web console] or the xref:../../virt/virtual_machines/virt-accessing-vm-consoles.adoc#virt-accessing-rdp-console_virt-accessing-vm-consoles[command line].
+* Collecting `must-gather` data about xref:../../virt/logging_events_monitoring/virt-collecting-virt-data.adoc#virt-must-gather-usage-targeted-vm-data_virt-collecting-virt-data[virtual machines].
+// * Collecting virtual machine memory dumps. [link TBD. CNV-20256]
```

Comment:
Please remove the end punctuation from all of the add'l resources list items (in both lists) 

> Write lists so that all or none of the items start with complete sentences. If list items comprise only complete sentences, include a period after each sentence. Except in definition lists, do not include any end punctuation if list items comprise only sentence fragments.
> If a list contains list items that start with sentence fragments, and one or more of the fragments are followed by complete sentences, include periods after all sentence fragments and sentences.

ISG: https://www.ibm.com/docs/en/ibm-style?topic=format-lists#unordered-lists